dace/transformation/dataflow/hbm_transform.py

```diff
@@ -0,0 +1,198 @@
+from typing import Any, Dict, Iterable, List, Tuple, Union
+
+import networkx
+from dace import dtypes, properties, registry, subsets, symbolic
+from dace.sdfg import utils, graph
+from dace.codegen.targets import fpga
+from dace.transformation import transformation, interstate
+from dace.sdfg import nodes as nd
+from dace import SDFG, SDFGState, memlet
+
+
+@registry.autoregister
+@properties.make_properties
+class HbmTransform(transformation.Transformation):
+    # type=List[Tuple[str, str, str]]
+    update_array_banks = properties.Property(
```

Comment:
I cannot see that you use this anywhere. Does it need to be part of this transformation?

Reply:
Yes definitively. It's not used for Axpy because that's such a simple transformation, but for more complex stuff it's quite usefull

Reply:
I need to understand how this transformation should be used in more detail first to understand why this should be here